test/XRP/wallet.test.ts

```diff
@@ -5,7 +5,8 @@ import 'mocha'
 
 /**
  * A mapping of input and expected outputs for BIP39 and BIP44.
- * @see https://iancoleman.io/bip39/#english
+ *
+ * @see [BIP39 - Mnemonic Code Converter]{@link https://iancoleman.io/bip39/#english}
```

Comment:
The `@see` tag is kind of weird in JSDoc and it's nondeterministic what will happen. TSDoc recommends using `@see` with the `@link` tag to be clear about what is going on.